docker/Dockerfile

```diff
@@ -447,6 +450,10 @@ COPY ./test_startup.sh /usr/libexec/pyrex/startup.d/
 # Precompile python files for improved startup time
 RUN python3 -m py_compile /usr/libexec/tini/*.py
 
+# Remove all non-root users and groups so that there are no conflicts when the
```

Comment:
That's a rather bare-metal way of doing it. How hard would it be to do a loop like (pseudocode):

  while [`getent passwd | wc -l` -gt 1 ]; do
     userdel ...
  done